docs/HelpWantedProcess.md

```diff
@@ -10,6 +10,14 @@ This process (we'll refer to it as `help-wanted` process going forward) triggers
 3. As part of filling out the template, the engineer will also need to **describe a high-level design for how to approach the problem and how it can be solved**.
 4. When all the sections of the template are filled in, the engineer will then need to **apply one of the `Complexity: <value>` labels**, to indicate how easy / hard will it be to tackle the issue.
  This will help community members to find the right type of issue to contribute to.
-5. After posting this comment, the assigned **engineer will unassign themself** from the issue and will **replace the `help-candidate` label with `help-wanted` label**, as an indicator, that the issue is ready for the community to pick up.
+5. After posting this comment the assigned engineer should:
+   - Unassign themself from the issue.
+   - **Replace the `help-candidate` label with `help-wanted` label**, as an indicator, that the issue is ready for the community to pick up.
```

Comment:
Nit, it's actually "help wanted" with a space, as that's github standard and some website aggregates such issues..

Reply:
Good catch. Thanks!